cluster/saltbase/salt/kube-apiserver/default

```diff
@@ -27,6 +27,10 @@
 {% if grains.cloud == 'gce' -%}
   {% set cloud_provider = "--cloud_provider=gce" -%}
 {% endif -%}
+{% if grains.cloud == 'aws' -%}
+  {% set cloud_provider = "--cloud_provider=aws" -%}
+  {% set cloud_config = "--cloud_config=/etc/aws.conf" -%}
```

Comment:
Sorry, my bad!
On Jan 30, 2015 9:05 AM, "Derek Carr" <notifications@github.com> wrote:

> In cluster/saltbase/salt/kube-apiserver/default
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3970#discussion_r23855516>
> :
>
> > @@ -27,6 +27,10 @@
> >  {% if grains.cloud == 'gce' -%}
> >    {% set cloud_provider = "--cloud_provider=gce" -%}
> >  {% endif -%}
> > +{% if grains.cloud == 'aws' -%}
> > +  {% set cloud_provider = "--cloud_provider=aws" -%}
> > +  {% set cloud_config = "--cloud_config=/etc/aws.conf" -%}
>
> this broke Vagrant.
>
> In the future, please always set the initial value of a global variable to
> the empty string outside of an if-block.
>
> I think this would have also broke other providers.
>
> Will send a patch.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3970/files#r23855516>
> .
>